your-rents-core/src/main/java/com/yourrents/core/dto/Property.java

```diff
@@ -20,14 +20,23 @@
  * #L%
  */
 
+import lombok.AllArgsConstructor;
 import lombok.Builder;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 import lombok.ToString;
 
 @Builder
 @Getter
 @ToString
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(of = {"id"})
 public class Property {
+    private Integer id;
```

Comment:
Do you like to expose the id in the external layer? I would like more to use the external_id.